### PR TITLE
There are cases where I want to use keys other than the fixed ones.

### PR DESCRIPTION
**Pull request by pyama86**
I've made adjustments because there are cases where I want to block not just based on the host, but also on IP for limiters of GET parameters and request paths.

### CODE REVIEW

File: base_limiter.go [hunk 1]
```diff
@@ -58,3 +59,19 @@ func (l *BaseLimiter) isTargetExtensions(r *http.Request) bool {
 	}
 	return false
 }
+func validateKey(key string) error {
+	for _, k := range []string{"remote_addr", "host"} {
```
Comment (k1LoW) [inline base_limiter.go]: "remote_addr" and "host" may be const variables.

**Review by k1LoW — APPROVED** [review]
LGTM.

We need to choose host or remote_addr, right?

File: base_limiter.go (hunk 1 above)
Comment (pyama86) [reply base_limiter.go]: I see.
I should to make constaiant variable host and remote_addr.

I will do afer cristmas..

**Comment by pyama86** [comment]
> We need to choose host or remote_addr, right?.

Yes!!!
I think that you are always right!!!

File: base_limiter.go [hunk 2]
```diff
@@ -10,6 +11,11 @@ import (
 	"github.com/2manymws/rl/counter"
 )
 
+const (
+	remoteAddrKey = "remote_addr"
+	hostKey       = "host"
```
Comment (k1LoW) [inline base_limiter.go]: How about exporting them and having users use them?

```suggestion
	RemoteAddrKey = "remote_addr"
	HostKey       = "host"
```

(Perhaps variable names when exporting might need to be considered.)

**Review by k1LoW — APPROVED** [review]
LGTM! Thank you!!!

**Pull request merged**